tests/bootstrap.php

```diff
@@ -7,3 +7,7 @@
 require_once __DIR__ . '/../vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php';
 require_once __DIR__ . '/../vendor/php-stubs/woocommerce-stubs/woocommerce-packages-stubs.php';
 require_once __DIR__ . '/../deps/scoper-autoload.php';
+
+if ( ! defined( 'HOUR_IN_SECONDS' ) ) {
```

Comment:
include this bootstrap:

phpstan/vendor/szepeviktor/phpstan-wordpress/bootstrap.php